lib/mixlib/shellout/windows.rb

```diff
@@ -107,12 +107,19 @@ def run_command
                 # Kill the process
                 if (Time.now - start_wait) > timeout
                   begin
+                    require 'wmi-lite/wmi'
+                    wmi = WmiLite::Wmi.new
+                    Utils.kill_process_tree(process.process_id, wmi, logger)
```

Comment:
This actually already does that. It just doesn't go to the logger but it raises an error - which is actually even louder.